lib/screens/settings/widgets/settings_muted_words.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_mobx/flutter_mobx.dart';
+import 'package:frosty/screens/settings/stores/settings_store.dart';
+import 'package:frosty/widgets/alert_message.dart';
+import 'package:mobx/mobx.dart';
+
+class SettingsMutedWords extends StatefulWidget {
+  final SettingsStore settingsStore;
+  const SettingsMutedWords({super.key, required this.settingsStore});
+
+  @override
+  State<SettingsMutedWords> createState() => _SettingsMutedWordsState();
+}
+
+class _SettingsMutedWordsState extends State<SettingsMutedWords> {
+  late final SettingsStore settingsStore;
+  final TextEditingController textController = TextEditingController();
+  final FocusNode textFieldFocusNode = FocusNode();
+
+  @override
+  void initState() {
+    settingsStore = widget.settingsStore;
+    super.initState();
+  }
+
+  void addMutedWord(String text) {
+    settingsStore.mutedWords.add(text);
```

Comment:
With the change in settings store, Mobx should be able to detect changes if we do something like this:

```suggestion
    settingsStore.mutedWords = [...settingsStore.mutedWords, text];
```